doc/modules/array_api.rst

```diff
@@ -123,7 +123,9 @@ Metrics
 - :func:`sklearn.metrics.pairwise.additive_chi2_kernel`
 - :func:`sklearn.metrics.pairwise.chi2_kernel`
 - :func:`sklearn.metrics.pairwise.cosine_similarity`
+- :func:`sklearn.metrics.pairwise.euclidean_distances`
 - :func:`sklearn.metrics.pairwise.paired_cosine_distances`
+- :func:`sklearn.metrics.pairwise.rbf_kernel`
```

Comment:
Maybe we should document the numerical stability expectations on devices that only allow `xp.float32` computation by reusing the section proposed in #27113.

Reply:
Sure.

Reply:
Considering that is already a part of another PR maybe we can leave it for now?

Reply:
Added the section.

Reply:
Please also refer to that section from bullet point for functions or estimators that have device dependent numerical precision handling.